src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,276 @@
+package duke.tasks;
+
+import java.util.ArrayList;
+
+import duke.exceptions.*;
+
+
+/**
+ * Add the ability to store whatever text entered by the user and display them back to the user when requested.
+ */
+
+public class TaskList {
+    private static ArrayList<Task> taskList = new ArrayList<>();
+
+    /**
+     * Commands
+     */
+    public static final String LIST = "list";
+    public static final String DONE = "done";
+    public static final String TODO = "todo";
+    public static final String DEADLINE = "deadline";
+    public static final String EVENT = "event";
+
```

Comment:
I believe these were defined in another class too? Consider making them final variables in the Command class so you can just create an instance of command and utilise them from there.